qa/tasks/util/rados.py

```diff
@@ -55,7 +55,7 @@ def create_cache_pool(remote, base_name, cache_name, pgnum, size, cluster_name="
     ])
     if application:
         remote.run(args=[
-            'sudo', 'ceph', 'osd', 'pool', 'application', 'enable', name, application, '--cluster', cluster_name
+            'sudo', 'ceph', 'osd', 'pool', 'application', 'enable', base_name, application, '--cluster', cluster_name
```

Comment:
i think it should be `cache_name`, otherwise we are tagging the base pool repeatedly, for example, in `rgw.py`'s `create_pools()`,

Reply:
@tchaikov thank you, fixed